sktime/benchmarking/_lib_mini_kotsu/run.py

```diff
@@ -58,16 +59,22 @@ def run(
         results_df = pd.DataFrame(columns=["validation_id", "model_id", "runtime_secs"])
         results_df["runtime_secs"] = results_df["runtime_secs"].astype(int)
 
+    from tqdm import tqdm
+
     results_df = results_df.set_index(["validation_id", "model_id"], drop=False)
     results_list = []
 
-    for validation_spec in validation_registry.all():
+    print("Printing Validations...")
+    print("\n")
+    for validation_spec in tqdm(validation_registry.all()):
         if validation_spec.deprecated:
             logger.info(
                 f"Skipping validation: {validation_spec.id} - as is deprecated."
             )
             continue
-        for model_spec in model_registry.all():
+        print("Running models...")
+        print("\n")
+        for model_spec in tqdm(model_registry.all()):
```

Comment:
I would propose to set `leave=False` to the second loop. This would remove the inner progress bar after the loop is finished. This would avoid a wall of loops and the user has not to scroll up to see the outer loop.